doc/i2p.md

```diff
@@ -47,9 +47,26 @@ In a typical situation, this suffices:
 bitcoind -i2psam=127.0.0.1:7656
 ```
 
-The first time Bitcoin Core connects to the I2P router, its I2P address (and
-corresponding private key) will be automatically generated and saved in a file
-named `i2p_private_key` in the Bitcoin Core data directory.
+The first time Bitcoin Core connects to the I2P router, if
+`-i2pacceptincoming=1`, then it will automatically generate a persistent I2P
```

Comment:
should we specify here what is default?

Reply:
On one hand that would be useful, on the other hand if the source code is changed and this forgotten, they would go out of sync.

This PR has 4 ACKs. Should I open a followup PR with this change after this PR is merged or leave it as is?

Reply:
You are right, this is certainly an insufficient reason to invalidate 4 acks.